### ISSUE TITLE
False positive for MD026 and abbreviations

### ISSUE THREAD

**Issue by JimDabell** [DavidAnson/markdownlint#1765]
When a heading includes a list and ends in “etc.”, this triggers MD026. However the period terminates the abbreviation for “et cetera” not the heading. It’s incorrect to omit the period after “etc”.

Correct:

```markdown
# Foo, Bar, Baz, etc.
```

Incorrect:

```markdown
# Foo, Bar, Baz, etc
```

**Comment by DavidAnson** [comment]
This is not a false positive; the heading ends with one of the forbidden punctuation characters. You happen to want to allow it in this case because it has semantic meaning.

I don't think it will be practical to try to list all of the possible abbreviations for this rule to try to detect and allow, so I recommend that you disable the rule for that part of the file or perhaps configure it to allow the period character if abbreviations are common in your project's headings.

**Comment by ssokolow** [comment]
Given how many markdown processors have no option to strip HTML comments, I'd really prefer a solution more along the lines of how I can put a `doc-valid-idents` key in my `clippy.toml` in Rust projects to have a single, project-wide place to say "These specific cases (eg. `etc.`) are always acceptable".

Some kind of exceptions configuration key for MD026 that could be specified once for the entire document in an out-of-the-way place like the end of the file.
